pkg/graveler/committed/merge_test.go

```diff
@@ -586,8 +587,216 @@ func Test_merge(t *testing.T) {
 					rng:    committed.Range{ID: "base:k3-k6", MinKey: committed.Key("k3"), MaxKey: committed.Key("k6"), Count: 2, EstimatedSize: 1234},
 				},
 			},
-			expectedSummary: graveler.DiffSummary{Count: make(map[graveler.DiffType]int)},
-			expectedErr:     graveler.ErrNoChanges,
+			expectedErr: nil,
+		},
+		"source and dest change same range": {
```

Comment:
```suggestion
		"dest removed range and added range after source removed range edges": {
```

Reply:
Not sure this is the best name, but it's a bit of a weird test, it's not that both changed the same range. It's that source changes are included in dest ranges.

Reply:
Done